inductiva/users/methods.py

```diff
@@ -0,0 +1,39 @@
+"""Methods to interact with the user info on the API."""
+import json
+from typing import Any, Dict, List
+
+import inductiva
+from inductiva import api
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.users_api import UsersApi
+
+
+def _fetch_quotas_from_api() -> List[Dict]:
+    """Get information about a user's quotas.
+    """
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = UsersApi(client)
+        try:
+            resp = api_instance.get_user_quotas().response
+
+            response_body = json.loads(resp.data.decode("utf-8"))
+
+            return response_body
+
+        except ApiException as e:
+            raise e
+
+
+def get_quotas() -> Dict[str, Dict[str, Any]]:
+    """Get the user quotas.
+
+    This function gets a dict with the user quotas.
+
+    Returns:
+        Dict with the user quotas.
+    """
+    quotas = _fetch_quotas_from_api()
+
+    return quotas
```

Comment:
This function is equivalent to:

```python
def get_quotas():
    return _fetch_quoatas_from_api()
```

Which is equivalent to `_fetch_quoatas_from_api`. Therefore why not just have all the code above here instead?

Reply:
We get an array of dicts from the back-end `[{name:"num_machines"},{name:"vcpus"},...]`. Right now, we just return that.

But I figure that in the future, it would be better to convert the array of dicts into a dict of dicts (since it makes it easier for the end user to manipulate the data).
```
{
      "vcpu":{...}
      "num_machines":{...}
}
```
So, in the future, that conversion will be placed in the get_quotas function.